.gitignore

```diff
@@ -24,6 +24,7 @@ build/working_dir
 
 # YAML for testings
 config*.yaml
+config*.yaml.bak
```

Comment:
I'm not very in favor of ignoring this if a `config.yaml.bak` needs to be excluded, then what should I do if I add a `config.yaml.bak2`? 
Maybe we can:
1. Exclude `config*.yaml*`; or
2. Manually avoid *.bak being `git add`.